### PR TITLE
add first day guide

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**Changes**
Add a First Day Programme with sections
- Intro - including schedule - took this from our initial discussion so needs review. @lgriffin 
- Buddy - needs filling out
- Buidling - not sure what to put in here. maybe Aoife can help use here.

### CODE REVIEW

File: first-day/intro/index.html [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+  <h1>Introduction</h1>
+
+
+
+
+  <h1>Schedule</h1>
+
+<h2>Week 1: Induction</h2>
+
+<p>Site walk around with Aoife for health and safety</p>
```
Comment (lgriffin) [inline first-day/intro/index.html]: Let's set this as a meeting with your mentor at 09:00 to 10:00 kind of thing. We get them, grab a cup of tea, learn a bit about them. Site walk can be at 10:00 to 10:15 kind of thing. Let's loop Aoife in here

File: first-day/intro/index.html [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+<p>Meet your buddy</p>
+
+<p>Laptop setup: Setup a Gmail account for gdocs and a Github Account</p>
```
Comment (lgriffin) [inline first-day/intro/index.html]: This sounds good, let's find hardware tomorrow!

File: first-day/intro/index.html [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+
+<p>Presentation and walkthrough of the goals of the placement, the rules enforced (homework, coding club, diary, Coder Dojo etc.)</p>
```
Comment (lgriffin) [inline first-day/intro/index.html]: We probably want a small presentation for this which we can link here

File: first-day/intro/index.html [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+
+<p>Introduction to MarkDown </p>
+
+<p>First Blog Post of your work experience in Markdown</p>
+
+<p>Brain Storm of features for the School Website in Markdown</p>
+
+<p>Homework: Collaboratively write a guide in MarkDown for the Schools Coding Club</p>	
+
+<h2>Week 3:</h2>
```
Comment (lgriffin) [inline first-day/intro/index.html]: I would halt this at Week 2 as the rest are more our scattered thoughts. I'd only be showing them a max of a week ahead at a time tbh

**Review by lgriffin — APPROVED** [review]

**Comment by damienomurchu** [comment]
LGTM, opened it & renders fine, good starting point :+1:

**Review by damienomurchu — APPROVED** [review]
LGTM, reviewed visually, good starting point. Good to merge to me :+1:

**Review by JameelB — APPROVED** [review]

**Pull request merged**